### ISSUE TITLE
Marathon LB under CentOS vs Debian

### ISSUE THREAD

**Issue by rcsavage** [mesosphere/marathon-lb#235]
Will you release a marathon-lb docker container that is running under CentOS rather than Debian?

**Comment by rcsavage** [comment]
I tried porting marathon-lb to centos, and it appears when I remove apps from the Marathon rest API, they are causing a problem during redeployments... in other words, when I redeploy the same app, the marathon-lb (in Centos) will not re-establish the "service" listener port.

**Comment by rcsavage** [comment]
It seem that runsv / runit / sv - does not play nice under CentOS compared to Debian.

**Comment by h0tbird** [comment]
What about Alpine Linux? The final image size should be much smaller.

**Comment by robsonpeixoto** [comment]
Have you configured the SELinux, @rcsavage ?

**Comment by rcsavage** [comment]
I have tried with SELinux enabled and disabled.   The problem has nothing to do with SELinux.   I have it working in CentOS now, but I have had to refactor the configuration and scripts used by marathon-lb.   The problem now is maintaining a fork of this repo, while it's not technically challenging, it's just extra work and time I should not have to be involved with.   It would be really nice that marathon-lb would have OS agnostic dependencies, and not tied to just Debian environments.
